src/middlewared/middlewared/plugins/vm/supervisor/supervisor_base.py

```diff
@@ -248,6 +247,11 @@ def get_domain_children(self):
             *self.commandline_xml(),
         ]
 
+        if self.vm_data.get('pin_vcpus') and self.vm_data.get('cpuset'):
```

Comment:
Why `.get`? (same question in other places too)

Reply:
It's long time since I did some python development but since those are defined as null=true I was thinking that could accessing those possibly lead into some KeyErrors if `.get` is not used?